Sources/Misc/SystemInfo.swift

```diff
@@ -40,7 +40,7 @@ class SystemInfo {
     let responseVerificationMode: Signing.ResponseVerificationMode
     let dangerousSettings: DangerousSettings
     let clock: ClockType
-    var preferredLocalesProvider: PreferredLocalesProviderType
+    let preferredLocalesProvider: PreferredLocalesProvider
```

Comment:
Since `preferredLocalesProvider` is now of type `PreferredLocalesProvider`, which is a class, this can now be a `let`